
Patch was sent upstream to Marcel Holtmann <marcel@holtmann.org>

Response:

Hi Marcin,

> I use Socket Bluetooth CompactFlash card in my Zaurus. Some time ago we 
> switched to 2.6.12-rcX-mmY so we get PCMCIA hotplug support. On my device 
> I also switched from pcmcia-cs 3.2.8 to pcmciautils 003 (will check 005 
> later).
> 
> With my config I lost automatic module loading after BT card insert so I 
> looked into kernel driver and added infos about my card. Patch attached.

there are some pending patches from Dominic that I want to see merged
mainline first, before I am going to submit patches for the Bluetooth
PCMCIA drivers.

> BTW - any plans on adding suspend/resume support into this driver? Now I 
> have to eject/insert card after resume and restart bluez-utils to get it 
> working.

Maybe with the new PCMCIA subsystem this will be possible. Feel free to
send in patches.

Regards

Marcel

--- linux-2.6.12/drivers/bluetooth/dtl1_cs.c.orig	2005-07-05 11:19:50.000000000 +0200
+++ linux-2.6.12/drivers/bluetooth/dtl1_cs.c	2005-07-05 11:19:51.000000000 +0200
@@ -810,6 +810,7 @@
 static struct pcmcia_device_id dtl1_ids[] = {
 	PCMCIA_DEVICE_PROD_ID12("Nokia Mobile Phones", "DTL-1", 0xe1bfdd64, 0xe168480d),
 	PCMCIA_DEVICE_PROD_ID12("Socket", "CF", 0xb38bcc2e, 0x44ebf863),
+	PCMCIA_DEVICE_PROD_ID12("Socket", "CF+ Personal Network Card", 0xb38bcc2e, 0xe732bae3),
 	PCMCIA_DEVICE_NULL
 };
 MODULE_DEVICE_TABLE(pcmcia, dtl1_ids);
